### PR TITLE
PD-1654 / 25.04 / Pd 1654 document cloud sync filename encryption known issue

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: static/includes/FilenameEncryption.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+&NewLine;
+
+{{< hint type=warning title="Filename Encryption"  >}}
+The rclone project has identified known issues with **Filename Encryption** in certain configurations, such as when long file names are used. See [SSH_FX_BAD_MESSAGE when syncing files with long filename to encrypted sftp storage](https://forum.rclone.org/t/ssh-fx-bad-message-when-syncing-files-with-long-filename-to-encrypted-sftp-storage/46228).
```
Comment (micjohnson777) [inline static/includes/FilenameEncryption.md:4]: two issues here, file names is two word and all caps for SFTP
```suggestion
The rclone project has identified known issues with **Filename Encryption** in certain configurations, such as when long file names are used. See [SSH_FX_BAD_MESSAGE when syncing files with long file names to encrypted SFTP storage](https://forum.rclone.org/t/ssh-fx-bad-message-when-syncing-files-with-long-filename-to-encrypted-sftp-storage/46228).
```

File: static/includes/FilenameEncryption.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+In some cases, this can prevent backup jobs from completing or being restored.
+
+We do not recommend enabling **Filename Encryption** for any cloud sync tasks that did not previously have it enabled.
+Users with existing cloud sync tasks that have this setting enabled must leave it enabled on those tasks to be able to restore those existing backups.
+Do not enable file name encryption on new cloud sync tasks
```
Comment (micjohnson777) [inline static/includes/FilenameEncryption.md:9]:
```suggestion
Do not enable file name encryption on new cloud sync tasks!
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
One sentence is bothering me but not enough to recommend changes. Found two other small issues

File: static/includes/FilenameEncryption.md (hunk 1 above)
Comment (DjP-iX) [reply static/includes/FilenameEncryption.md:4]: This one is just the title of the forum post, so I don't think we should edit it, even if we'd quibble with the details

File: content/SCALE/GettingStarted/SCALEReleaseNotes.md [hunk 3]
```diff
@@ -135,7 +135,7 @@ Notable changes:
 
 ### 25.04-BETA.1 Known Issues
 
-* 
+* An issue has been discovered for cloud sync tasks configured with **Filename Encryption**, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with **Filename Encryption** enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```
Comment (micjohnson777) [inline content/SCALE/GettingStarted/SCALEReleaseNotes.md:138]: I missed this the first time, in this case you are talking about the feature, like saying  replication, so Filename Encryptions as the UI element is not correct here unless specifically talking about this in context, this is more a general mention. so I recommend either of these options:
```suggestion
* An issue has been discovered for cloud sync tasks configured using the **Filename Encryption** setting, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with the **Filename Encryption** setting enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```
or
```suggestion
* An issue has been discovered for cloud sync tasks configured with file name encryption, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with the **Filename Encryption** setting enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
sorry, I found something else.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**